Xamarin.Essentials/Connectivity/Connectivity.ios.cs

```diff
@@ -13,6 +14,22 @@ static void StartListeners()
             listener.ReachabilityChanged += OnConnectivityChanged;
         }
 
+        static SignalStrength PlatformSignalStrength()
+        {
+            var signalstrength = (int)new NEHotspotNetwork().SignalStrength * 3;
```

Comment:
According to the docs, the range is **0.0 - 1.0**. This formula is mostly ok, but really looses the potential for 0.99 to be marked as strong because you cast to int.

I am thinking that maybe just do: 0 = none, 0 - 0.3 = weak, 0.3 - 0.7 = fair and 0.7+ = strong.

Reply:
Yeah, I did not put the Math.Round call in before the int cast. I agree with your splitting of the range, but as we are working with double values I changed everything below 0.01 to be None. This should be fine as well and lets us keep our nice math :)